bundle-analyzer.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  files: [
+    {
+      test: '*.js',
+      maxSize: '200 kB',
+    },
+    {
+      test: '*.css',
+      maxSize: '100 kB',
```

Comment:
@koss-lebedev Nothing but just out of curiosity - How did we come up with these `maxSize` values? Is it a standard? Thanks

Reply:
@HonzaBartos Well, the default setting for Bundle Analyzer is to cap all files at 250kb. I checked that the output from Gatsby for JS was under 200kb, so I thought it'd be nice to keep it that way 🙂The value for CSS size is random, just from experience, I think it should never grow over 100kb

Reply:
Afaik a standard for JavaScript is 170kB :) 

I would leave there a note/todo, that this supposed to be ideally set in hand with performance budget. Even though this one kinda behaves as a perf budget, but it's not exactly it :)